microsoft/electionguard-ui
# 🗳  Show Elections

[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue

#87 

### Description
Updates the election list page to show a list of elections, which it pulls from an API call to the back-end.  This borrows heavily from `ElectionTable.tsx`, however that didn't fit the data, so this starts over.

### Testing

1. Ensure the back-end is running
2. Add an election to the back end through postman
3. In the UI click manage elections

You should see something like this:

![image](https://user-images.githubusercontent.com/1769905/149047471-361e6355-0154-4698-9785-4bc4979b7766.png)

If there are lots of elections it will support pagination like this:

![image](https://user-images.githubusercontent.com/1769905/149047942-4b2bb27d-7623-4810-bafc-de4615dd6865.png)

It supports filtering:

![image](https://user-images.githubusercontent.com/1769905/149048002-74976f85-029a-4cd6-9ba7-c4496bc2812f.png)

And sorting:

![image](https://user-images.githubusercontent.com/1769905/149048061-fdff5575-d342-440c-93f8-083c2f470d50.png)
